packages/terra-search-field/tests/nightwatch/search-field-spec.js

```diff
@@ -0,0 +1,30 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-search-field', done);
+  },
+
```

Comment:
Will do, I was a little bit confused about what to test with jest vs nightwatch.  I will add these in.